internal/services/manager_test.go

```diff
@@ -59,6 +60,12 @@ func TestRegisterGRPCServices(t *testing.T) {
 	defer require.NoError(t, m.Stop(), "Teardown: Stop should not have returned an error, but did")
 
 	got := m.RegisterGRPCServices(context.Background()).GetServiceInfo()
+	// Make the content of the golden file deterministic by sorting the methods by name.
+	for _, info := range got {
+		sort.Slice(info.Methods, func(i, j int) bool {
```

Comment:
Good. I'd just (nit) use `slices.SortFunc` though, since as the doc says: 

> // Note: in many situations, the newer [slices.SortFunc] function is more
// ergonomic and runs faster.

Reply:
done